lib/components/viewers/stop-schedule-table.tsx

```diff
@@ -1,5 +1,7 @@
-import { FormattedMessage, FormattedTime } from 'react-intl'
-import moment from 'moment'
+import { FormattedMessage } from 'react-intl'
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore TYPESCRIPT TODO: wait for typescripted core-utils
```

Comment:
I believe we can remove this comment, right? We've upgraded core-utils

Reply:
Apologies -- this has been done in the batch ui redesign, but I forgot that that hasn't been merged yet.